docs/examples/storage/preauthrequests/preauthreq.tf

```diff
@@ -20,8 +20,6 @@ variable "bucket_name" {}
 variable "object_name" {}
 
 provider "baremetal" {
-  region = "us-phoenix-1"
-  alias = "phx"
```

Comment:
If you define a provider with an alias, resources must specify that alias, otherwise it appears terraform is attempting to instantiate its own bmcs provider, which would be why it then prompts for all the required params but fails because both key path and key material appear to be optional but in reality one or the other is required.